kythe/java/com/google/devtools/kythe/platform/shared/Metadata.java

```diff
@@ -32,6 +32,14 @@
  * path to a loadable metadata file.
  */
 public class Metadata {
+  /** An additional semantic to apply to the given (java) node. */
+  public enum Semantic {
```

Comment:
@zrlk I added a separate enum here rather than re-using the proto-defined one since the c++ implementation did the same. Is this necessary? The only advantage I think for java is that we're not using ALIAS, so that's omitted from the java-defined enum.

Reply:
Sounds good, done.